src/lib/db/query.js

```diff
@@ -2,42 +2,36 @@
 
 const db = require('./postgres');
 
-const countsByType = 'SELECT dataset, count(*)::INTEGER FROM lev_audit WHERE date_time > $(from)';
-const countsByUser =
-  'SELECT date_time::DATE AS date, dataset, username, count(*)::INTEGER FROM lev_audit WHERE date_time > $(from)';
-const until = 'AND date_time < $(to)';
-const groupByType = ' GROUP BY dataset';
-const groupByDateTypeUser = ' GROUP BY date_time::date, dataset, username ORDER BY date_time::date';
 const groupByTypeGroup = 'GROUP BY name, dataset';
 const totalCount = 'SELECT count(*)::INTEGER FROM lev_audit';
-const forToday = ' WHERE date_time::DATE = current_date';
-const fromDate = 'date_time::DATE >= $(from)';
-const toDate = 'date_time::DATE < $(to)';
+const forToday = ' WHERE date_time >= (current_date::date)::timestamp';
+const fromDate = 'date_time >= $(from)';
+const toDate = 'date_time < $(to)';
 const searchGroup = 'groups::TEXT ILIKE \'%\' || $(group) || \'%\'';
 
 const buildCountsByGroup = (from, to, includeNoGroup = true) => `
```

Comment:
Are `from` and `to` still being used?

Reply:
Yep...also in some of the work that I'm doing on removing groups.

Reply:
I've refactored this now anyway. The new version is more in keeping with the other SQL queries, and should be easier for changes to be made to it in the future...